docs/dev/libvirt/customization.md

```diff
@@ -20,3 +20,45 @@ platform:
 pullSecret: '{"auths": ...}'
 sshKey: ssh-ed25519 AAAA...
 ```
+
+## Change cpu/memory
+
+- Create manifests
+
+```bash
+$ openshift-install --log-level=debug --dir=. create manifest
+```
+
+- Edit openshift/99_openshift-cluster-api_master-machines-0.yaml to read (only relevant part shown)
+
+```yaml
+apiVersion: machine.openshift.io/v1beta1
+kind: Machine
+spec:
+  providerSpec:
+    value:
+      apiVersion: libvirtproviderconfig.openshift.io/v1beta1
+      domainMemory: 8192
+      domainVcpu: 4
+```
+
+- Edit openshift/99_openshift-cluster-api_worker-machineset-0.yaml to read (only relevant part shown)
+
+```yaml
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+spec:
+  template:
+    spec:
+      providerSpec:
+        value:
+          apiVersion: libvirtproviderconfig.openshift.io/v1beta1
+          domainMemory: 16384
+          domainVcpu: 6
+```
+
```

Comment:
Why show both files? should I always set worker memory to twice the memory of the master?
You could explicitly say that first one is to change the memory/vcpus for the master nodes, and that the 2nd one is for the workers, and give some guidelines for the minimal values to use. Mention that memory is in kB